### DISCUSSION TITLE
[Feat]: Autoplay - Possibility to create a progress bar

### DISCUSSION THREAD

**Discussion by tyteen4a03** [davidjerleke/embla-carousel#1059] [Ideas] [answered]
### Summary

Is it possible to have individual progress bars for each slide, and have them play the progress during the delay?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Additional information

- PR: #1062.

**Comment by davidjerleke** [comment]
Thanks for your question @tyteen4a03.

At the time of writing there are hacky ways of doing this but this feature is on the roadmap:

- #841

**Reply by davidjerleke** [reply]
@tyteen4a03 it's now possible to create an autoplay progress bar. I've added it to [this example](https://www.embla-carousel.com/examples/predefined/#autoplay) in the docs.

**Comment by davidjerleke** [comment, marked as answer]
This is possible to achieve from [v8.5.0](https://github.com/davidjerleke/embla-carousel/releases/tag/v8.5.0) and up. See [example](https://www.embla-carousel.com/examples/predefined/#autoplay).
